src/duiker.py

```diff
@@ -36,7 +36,7 @@ def xdg_data_home(name: Optional[str] = None) -> pathlib.Path:
     Return the XDG Base Directory Specification data directory for a specific
     application, or the base directory itself.
     """
-    home = pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or '~/.local/share'))
+    home = pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or os.path.expanduser('~') + '/.local/share'))
```

Comment:
If you can find a way to reproduce this I'll gladly accept it. I wasn't able to reproduce this in the official CentOS 7 Docker container using Python 3.5.1 from SCL:

```
[test@59ba1ec4939a ~]$ python
Python 3.5.1 (default, Oct 21 2016, 21:37:19)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import pathlib, os
>>> os.environ.get('XDG_DATA_HOME')
>>> pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or '~/.local/share'))
PosixPath('/home/test/.local/share')
>>> pathlib.Path(os.path.expanduser(os.environ.get('XDG_DATA_HOME') or os.path.expanduser('~') + '/.local/share'))
PosixPath('/home/test/.local/share')
```